bookops_worldcat/metadata_api.py

```diff
@@ -556,32 +559,33 @@ def holdings_unset_multi_institutions(
 
         return query.response
 
-    def search_brief_bib_other_editions(
+    def search_brief_bibs_other_editions(
         self,
         oclcNumber: Union[int, str],
-        deweyNumber: Optional[str] = None,
-        datePublished: Optional[str] = None,
+        deweyNumber: Optional[Union[str, List[str]]] = None,
```

Comment:
Wonder if syntax like that is permissible:
```python
my_arg: Union[str, List[str], None]
```
or maybe `Optional` is clearer?

Reply:
I think they're equivalent and both permissible but the mypy documentation says `Optional` is preferred.